src/app/(main)/profile/myActivitiesStatus/page.tsx

```
@@ -120,42 +145,33 @@ export default function ReservationStatusPage() {
-
-            // apiReservationData에서 해당 날짜 데이터 가져오기
+            // 📋 fallback 데이터에서 해당 날짜 정보 추출
             const fallbackData = dashboardData[date];
             if (fallbackData && fallbackData.length > 0) {
-              console.log(`📋 ${date} fallback 데이터:`, JSON.stringify(fallbackData, null, 2));
-
-              // fallback 데이터에서 상태 정보 추출
+              // 🔍 fallback 데이터에서 상태 정보 추출 (중첩된 구조 처리)
               fallbackData.forEach((schedule) => {
                 if (schedule.reservations && Array.isArray(schedule.reservations)) {
                   (schedule.reservations as any[]).forEach((reservationGroup) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)